config/config_test.go

```diff
@@ -1,4 +1,4 @@
-// +build all race
+//go:build all || race
```

Comment:
I'd prefer to update all go build tags in the project at the same time. We can do that in a separate PR 👍🏻

Reply:
Please see my most recent commit f65b90c64a790eafb64ab2fe47f6742abbe7b164 in which I reverted that change. _go fmt_ isn't happy about it, but it will come around.

Reply:
Thanks - I agree we need to move the newer standard and will do the change package wide separately in another PR 👍🏻